docs/strategy-advanced.md

```diff
@@ -177,16 +178,33 @@ class AwesomeStrategy(IStrategy):
         return -0.15
 ```
 
+
+#### Calculating stoploss relative to open price
+
+Stoploss values returned from `custom_stoploss` always specify a percentage relative to `current_rate`.  In order to set a stoploss relative to the *open* price, we need to use `current_profit` to calculate what percentage relative to the `current_rate` will give you the same result as if the percentage was specified from the open price.
+
+This can be calculated as:
+
+``` python
+def stoploss_from_open(open_relative_stop: float, current_profit: float) -> float:
+    return 1-((1+open_relative_stop)/(1+current_profit))
+
+```
+
+For example, say our open price was $100, and `current_price` is $121 (`current_profit` will be `0.21`).  If we want a stop price at 7% above the open price we can call `stoploss_from_open(0.07, 0.21)` which will return `0.1157024793`.  11.57% below $121 is $107, which is the same as 7% above $100.
+
 #### Trailing stoploss with positive offset
 
 Use the initial stoploss until the profit is above 4%, then use a trailing stoploss of 50% of the current profit with a minimum of 2.5% and a maximum of 5%.
 
-Please note that the stoploss can only increase, values lower than the current stoploss are ignored.
 
 ``` python
 from datetime import datetime, timedelta
 from freqtrade.persistence import Trade
 
+def stoploss_from_open(open_relative_stop: float, current_profit: float) -> float:
+    return 1-((1+open_relative_stop)/(1+current_profit))
+
```

Comment:
Instead of copy/pasting the same function in 3 samples ... (which basically results in users having to copy/paste it to each strategy ...

should this maybe be added to the codebase so it can be imported (similar to `merge_informative_pair()`) ?

Reply:
For sure, I'll take a look.

Reply:
I've moved `stoploss_from_open` into a helper function, and updated the docs and tests accordingly.